paper/extract_triplets_tweets.py

```diff
@@ -2,24 +2,21 @@
 import os
 import random
 from pathlib import Path
-from typing import List, Optional, Generator, Union, Dict
-import spacy
+from typing import List, Optional, Generator, Union
+
 from spacy.tokens import Doc, Span
 import argparse
-import sys
 
 from data import load_gold_triplets
 import spacy
 from extract_examples import extract_examples
-from conspiracies.prompt_relation_extraction import (
+from conspiracies.docproc.relationextraction import (
```

Comment:
```suggestion
from conspiracies.spacy_component import (
```
I would format it to the following.

Reply:
Do you mean `spacy_component` instead of `docproc`? Or that components should be importable from the top-level package instead of having to go through the nested package structure?

Reply:
at least the first. The nested structure is not that important with the entry points (as you don't need to import it).